src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -127,6 +128,8 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
   @internalProperty()
   private _manifests: Record<string, IntegrationManifest> = {};
 
+  private _manifestFetched?: Set<string>;
```

Comment:
```suggestion
  private _fetchedManifests?: Set<string>;
```
Might be a purely subjective thing, but that var name makes it sound like a boolean flag, whereas for actual data I would have assumed `_fetchedManifests` (plural to show it contains multiple elements and more more natural word order).

Nothing critical, so feel free to ignore the comment.

Reply:
Renamed it to `_extraFetchedManifests`.

Reply:
That's even better 👍